Gemfile

```diff
@@ -51,3 +54,5 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem 'graphiql-rails', group: :development
```

Comment:
you can add this to the existing development group. lines 28 - 34

Reply:
Sorted!